public/app/percona/inventory/Tabs/Agents.tsx

```diff
@@ -51,8 +52,13 @@ export const Agents: FC<GrafanaRouteComponentProps<{ serviceId: string; nodeId:
   const { isLoading: nodesLoading, nodes } = useSelector(getNodes);
   const styles = useStyles2(getStyles);
 
+  const mappedNodes: Node[] = useMemo(
+    (): Node[] => nodeFromDbMapper(nodes).sort((a, b) => a.nodeName.localeCompare(b.nodeName)),
```

Comment:
Just a note, not an issue, be careful with the `sort` method of array, in difference to methods like `map/filter/reduce` it does **NOT** return a new array, but instead returns a reference to the original array. (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)